packages/at_commons/lib/src/verb/sync_verb_builder.dart

```diff
@@ -21,13 +21,10 @@ class SyncVerbBuilder implements VerbBuilder {
   @override
   String buildCommand() {
     StringBuffer serverCommandBuffer = StringBuffer('sync:');
-    if (isPaginated) {
-      serverCommandBuffer.write('from:');
-    }
+    // removed isPaginated logic, since sync:from is the default sync verb on server
+    serverCommandBuffer.write('from:');
     serverCommandBuffer.write('$commitId');
-    if (isPaginated) {
-      serverCommandBuffer.write(':limit:$limit');
-    }
+    serverCommandBuffer.write(':limit:$limit');
```

Comment:
Does the comment on line 15 mean that `limit` is ignored on the atServer? If so, then perhaps we shouldn't deprecate `limit` but instead respect it on the atServer side?

Reply:
in SyncProgressiveVerbHandler, only fromCommitId and regex are read from verb params. limit is not read.
There are two options
1) rely on default limit - 25 that is configured in SyncProgressiveVerbHandler --> commitLog.getEntries and deprecate limit param in sync verb builder
2) Keep limit param in sync verb builder and modify SyncProgressiveVerbHandler to read limit from verb params and pass the limit to commitLog.getEntries

Reply:
@murali-shris Option 2 please, and change the default in the SyncVerbBuilder to be in line with whatever the current default is on the server (25)

Reply:
Done.
Server PR:
https://github.com/atsign-foundation/at_server/pull/2182
Client PR:
https://github.com/atsign-foundation/at_client_sdk/pull/1450